app/helpers/link_helper.rb

```diff
@@ -0,0 +1,10 @@
+module LinkHelper
+  def inverse_link(text, path: nil, inverse: false)
```

Comment:
Could there be a better name here? This name makes it looks like a function which returns an inverse_link (rather than a function which returns a non-inverse link, or an inverse link if you say please).